src/data.js

```diff
@@ -20,20 +21,18 @@ module.exports = (router = new Router()) => {
   });
 
   router.post('/', (req, res) => {
-    const { files: { file } = {} } = req;
-    if (!file) {
-      return res.status(400).send('Collected data file was not uploaded!');
-    }
-    if (!file.name) {
-      return res.status(400).send('Collected data file missing name');
+    const jsonData = req.body
+    if (Object.keys(jsonData).length === 0) {
+      return res.status(400).send('Malformed or bad data');
     }
+    const jsonDataFile = jsonData['file']
```

Comment:
Can you make it so that if 'file' is not provided in the JSON, we fallback to just naming it the current time since epoch? I don't think the frontends should have to concern themselves with the names of the files.